ClientApp/app/tabs/_layout.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import FontAwesome from '@expo/vector-icons/FontAwesome';
+import { Tabs } from 'expo-router';
+import { useRouter } from 'expo-router';
+
+export default function TabLayout() {
+  const router = useRouter();
+
+  return (
+    <Tabs
+      screenOptions={{
+        tabBarActiveTintColor: '#0C9E04',
+        tabBarStyle: {
+          paddingHorizontal: 10,
+          height: 100,
+        },
+        tabBarLabelStyle: {
+          marginTop: -10,
+          fontSize: 12,
+          padding: 10,
+          textAlign: 'center',
+        },
+      }}
+    >
+      {/* Home Tab */}
+      <Tabs.Screen
+        name="home"
+        options={{
+          title: 'Home',
+          headerShown: false,
+          tabBarIcon: ({ color }) => <FontAwesome size={28} name="home" color={color} />,
+        }}
+      />
+      
+      {/* Plus Tab */}
+      <Tabs.Screen
+        name="plus"
```

Comment:
The name should probably be changed to something like "addEvent". The file name should also but changed too accordingly.

Reply:
Noted! This will be fixed in the next iteration! Thanks